docs/configuration/api.md

```diff
@@ -38,6 +41,16 @@ For more customization, see [entry points](/configuration/entrypoints/) document
 
 ![Web UI Health](/img/traefik-health.png)
 
+## Security
+
```

Comment:
Maybe a sentence about not exposing the api port?

Reply:
done, thanks for the idea and feedback!